src/core_plugins/kibana/public/management/app.html

```diff
@@ -13,11 +13,12 @@
       ></bread-crumbs>
 
       <!-- Tabs. -->
-      <div data-transclude-slot="bottomRow" class="kuiLocalTabs">
+      <div data-transclude-slot="bottomRow" class="kuiLocalTabs" role="tablist">
```

Comment:
Do we have issues to apply this role and the other accessibility improvements here to all of the kuiLocalTabs in OSS Kibana and X-Pack?

Reply:
Not individual ones I guess, but I can just open another PR right after this one for it.